deployer/README.md

```diff
@@ -1175,3 +1175,260 @@ the first created. You can check if the route in question is defined in multiple
     logging     kibana-ops   kibana-ops.example.com              logging-kibana-ops
 
 (In this example there are no overlapping routes.)
+
+## Splitting your Fluentd for high scale capacity (Advanced)
+
+In scenarios where the number of Fluentd nodes starts to exceed 100, the watching
+connection to Kubernetes that is used within Fluentd to collect pod metadata can
+cause the cluster to exceed the *maxRequestsInFlight* configuration setting. This
+can cause symptoms such as Fluentd being unable to connect to the Kubernetes service
+to collect metadata and preventing it from being able to process and ship logs.
+
+The alternative to setting *maxRequestsInFlight* to be unlimited, potentially creating
+a larger load on the cluster, is to split up the Fluentd deployments into collector
+and enricher pods (as opposed to all-in-one which is provided OOTB).
+
+This configuration leverages Fluentd reading its configuration from a configmap
+and the Fluentd image having the [fluent-plugin-secure-forward](https://github.com/tagomoris/fluent-plugin-secure-forward)
+plugin installed.
+
+First, unschedule all of Fluentd pods (Elasticsearch, Kibana, and Curator
+can all remain running):
+
+    $ oc label node --all logging-infra-fluentd-
+
+Where "logging-infra-fluentd" is the nodeSelector configured for the Fluentd daemonset.
+
+Next, create the configmap that will be used for the Enriching Fluentd.
+
+First, generate a random key that will be used for communication between the two
+types of Fluentd pods:
+
+    $ tr -dc 'a-zA-Z0-9' < /dev/urandom | head -c 128
```

Comment:
You might want to give this a shell variable name for the value, as a way to refer to it in the .conf file below.